pkg/pravega/pravega_controller.go

```diff
@@ -180,3 +214,37 @@ func makeControllerService(pravegaCluster *api.PravegaCluster) *corev1.Service {
 		},
 	}
 }
+
+func makeControllerPodDisruptionBudget(pravegaCluster *api.PravegaCluster) *policyv1beta1.PodDisruptionBudget {
+	var maxUnavailable intstr.IntOrString
+
+	if pravegaCluster.Spec.Pravega.ControllerReplicas == int32(1) {
```

Comment:
Can't we express this as min available of 1 rather than having two options for max unavailable? it is just to simplify the logic.

Reply:
Well, the logic will be different if we change it to `minAvailable=1`.

- `maxUnavailable=1` means that, regardless of the number of Controller replicas, only one of them can be disrupted. E.g. if we have 5 replicas, we can only disrupt 1.
- `minAvailable=1` means that we allow disruptions as long as we have one Controller replica. E.g. if we have 5 replicas, we allow 4 disruptions.

In the existing, we allow 1 planned disruption if we have more than 1 controller, 0 otherwise. Does this behavior look good to you?